ui/packages/consul-ui/package.json

```diff
@@ -117,6 +117,7 @@
     "ember-modifier": "^2.1.1",
     "ember-named-blocks-polyfill": "^0.2.3",
     "ember-on-helper": "^0.1.0",
+    "ember-on-resize-modifier": "^0.3.0",
```

Comment:
Interesting. We should replace `{{on-window 'resize' (action "resize") }}` with this.

Reply:
Yes definitely!